.github/PULL_REQUEST_TEMPLATE.md

```
@@ -12,6 +12,10 @@
      - 📗 Update any related documentation and include any relevant screenshots.
 -->
 
+## Preview Link
+
+https://submit-pr-[PR_NUMBER].herokuapp.com/
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)